tools/physiofit/physiofit.xml

```diff
@@ -0,0 +1,202 @@
+    #end if
+    #if '$input_selection.input_selector' == 'json':
+        --data '$input_selection.datafile'
+        --config '$input_selection.cfgfile'
+    #end if
+    '$basic_settings.lag'
+    #if '$basic_settings.deg_select.deg' == 'true':
+        -d '$basic_setting.degconsts'
+    #end if
+    #if '$basic_settings.mc_select.montecarlo' == 'true':
+        -mc '$basic_settings.mc_select.iters'
+    #end if
+
+    --vini '$advanced_settings.vini'
+
+    #if '$advanced_settings.sd_select.sd' == 'true':
+        -s '$advanced_settings.sd_select.stddevs'
+    #end if
+    #if '$advanced_settings.cm_select.cm' == 'true':
+        -cm '$advanced_settings.cm_select.conc_met_bounds'
+    #end if
+    #if '$advanced_settings.fm_select.fm' == 'true':
+        -fm '$advanced_settings.fm_select.flux_met_bounds'
+    #end if
+    #if '$advanced_settings.cb_select.cb' == 'true':
+        -cb '$advanced_settings.cb_select.conc_biom_bounds'
+    #end if
+    #if '$advanced_settings.fb_select.fb' == 'true':
+        -fb '$advanced_settings.fb_select.flux_biom_bounds'
+    #end if
+    #if '$advanced_settings.verbose':
+        -v
+    #end if
+    -op '$plots' -of '$fluxes' -os '$stats' -oc '$config'
+
+    ]]></command>
+    <inputs>
+
+        <conditional name='input_selection'>
+            <param name='input_selector' type='select' label='Choose the input data type to upload (tsv or json config file. For more information please check the documentation'>
+                <option value='tsv'>tsv file</option>
+                <option value='json'>config file (json)</option>
+            </param>
+            <when value='tsv'>
+                <param name='datafile' type='data' format='tsv, txt' multiple='true' label='Upload data file containing biomass and concentration values'/>
+            </when>
+            <when value='json'>
+                <param name='datafile' type='data' format='tsv, txt' multiple='true' label='Upload data file containing biomass and concentration values'/>
+                <param name='cfgfile' type='data' format='json' label='Upload configuration file' />
+            </when>
+        </conditional>
+
+
+        <section name='basic_settings' title='Basic Settings'>
+
+            <param name='lag' type='boolean' argument='--lag' label='Flag to estimate lag phase' truevalue='-l' falsevalue=''/>
+
+            <conditional name='deg_select'>
+                <param name='deg' type='boolean' argument='--deg' label='Flag to add degradation constants' truevalue='true' falsevalue=''/>
+                <when value='true'>
+                    <param type='text' name='degconsts' value='{}' label='Enter degradation constants in dictionary format (see docs for more information)'/>
+                </when>
+            </conditional>
+
+            <conditional name='mc_select'>
+                <param name='montecarlo' type='boolean' argument='--montecarlo' label='Flag for sensitivity analysis' truevalue='true'/>
```

Comment:
please use a select parameter here not boolean: https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#booleans